src/Compilers/CSharp/Test/Semantic/Semantics/UnsafeTests.cs

```diff
@@ -5540,7 +5540,7 @@ void M()
 
             compilation.VerifyDiagnostics();
 
-            foreach (var declarationSyntax in tree.GetCompilationUnitRoot().DescendantTokens().OfType<VariableDeclarationSyntax>().Where(syntax => syntax.GetFirstToken().IsKind(SyntaxKind.VoidKeyword)))
+            foreach (var declarationSyntax in tree.GetCompilationUnitRoot().DescendantNodes().OfType<VariableDeclarationSyntax>().Where(syntax => syntax.GetFirstToken().IsKind(SyntaxKind.VoidKeyword)))
```

Comment:
interesting.  seems like the old code could never have worked before...

Reply:
The old code contained assertions that never executed. This is one example of the type of error that code coverage can help prevent, provided you include code coverage data for test assemblies.
https://github.com/coverlet-coverage/coverlet/issues/618

Reply:
/cc @StingyJack @PureKrome for a real-world example of a situation that would have benefitted from code coverage on test code

Reply:
thanks @sharwell for the real-case.